### PR TITLE
Updated sample app to use/work with dev (future 6.0)

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

**Comment by wmathurin** [comment]
Allowed me to reproduce this issue: 
https://github.com/forcedotcom/SalesforceMobileSDK-iOS/issues/2319

And to come up with this fix:
https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2324

File: android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java [hunk 1]
```diff
@@ -24,55 +24,29 @@
...
 
-    /**
+	/**
 	 * Returns the name of the main component registered from JavaScript.
 	 * This is used to schedule rendering of the component.
 	 */
 	@Override
 	protected String getMainComponentName() {
-		return "MyUserPicReactNative";
+		return "ReactNativeTemplate";
```
Comment (bhariharan) [inline android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java]: Should be `MyUserPicReactNative`.

File: android/settings.gradle [hunk 2]
```diff
@@ -1,8 +1,11 @@
-rootProject.name = 'MyUserPicReactNative'
+rootProject.name = 'ReactNativeTemplate'
```
Comment (bhariharan) [inline android/settings.gradle]: `MyUserPicReactNative`.

File: js/UserPic.js [hunk 3]
```diff
@@ -89,7 +93,7 @@ module.exports = React.createClass({
     },
 
     getUserInfo(callback) {
-        forceClient.sendRequest('/services/data', '/v36.0/chatter/users/me', 
+        net.sendRequest('/services/data', '/v36.0/chatter/users/me',
```
Comment (bhariharan) [inline js/UserPic.js]: Should we switch this to `v41.0` to be inline with `6.0`?

File: js/UserPic.js [hunk 4]
```diff
@@ -103,7 +107,7 @@ module.exports = React.createClass({
     },
 
     uploadPhoto(localPhotoUrl, callback) {
-        forceClient.sendRequest('/services/data', '/v36.0/connect/user-profiles/' + this.state.userId + '/photo', 
+        net.sendRequest('/services/data', '/v36.0/connect/user-profiles/' + this.state.userId + '/photo',
```
Comment (bhariharan) [inline js/UserPic.js]: Should we switch this to `v41.0` to be inline with `6.0`?

**Review by bhariharan — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java (hunk 1 above)
Comment (wmathurin) [reply android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java]: I ran forcereact to regenerate the app - I will check if there is a bug there.

Comment (wmathurin) [reply android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java]: I ran forcereact to regenerate the app - I will check if there is a bug there.

File: js/UserPic.js (hunk 3 above)
Comment (wmathurin) [reply js/UserPic.js]: Will do.

File: android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java (hunk 1 above)
Comment (wmathurin) [reply android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java]: Done.

**Review by bhariharan — APPROVED** [review]

**Comment by bhariharan** [comment]
👍

**Pull request merged**